tests/integration/scripts/test_cat_log.py

```diff
@@ -0,0 +1,53 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Integration test for the cat log script.
+"""
```

Comment:
Can the tests in `tests/functional/cylc-cat-log/03-bad-workflow.t` be translated to here?

Reply:
On second thoughts may not be worth the time

Reply:
Would have been trivial if my VDI weren't being a pain.